testgrid/config/config.yaml

```diff
@@ -3506,16 +3526,24 @@ dashboards:
   dashboard_tab:
   - name: gce-device-plugin-gpu
     test_group_name: ci-kubernetes-e2e-gce-device-plugin-gpu
+  - name: gce-device-plugin-gpu-stable1
```

Comment:
Let's call this `gce-device-plugin-gpu-1-8` like the others.

Reply:
whoops, slipped, will patch up

Reply:
sneaked the fix in https://github.com/kubernetes/test-infra/pull/4689